xblock/test/test_runtime.py

```diff
@@ -524,10 +524,8 @@ def test_multiply_mixed(self):
         assert_equals(4, len(post_mixed.__bases__))
 
 
-@XBlock.needs("i18n")
-@XBlock.wants("secret_service")
-@XBlock.needs("no_such_service")
-@XBlock.wants("another_not_service")
+@XBlock.needs("i18n", "no_such_service")
+@XBlock.wants("secret_service", "another_not_service")
```

Comment:
This is an improvement - but it did take a test away that checks that multiple `wants` or `needs` don't overwrite each other. From the implementation, it's clear that they don't. But just pointing it out.